owenhallgren/stretch-project
# Feature/media queries

## What does this PR do?

- This adds media queries to our app. 
- It adds some icons for our nav bar to clean up the look
- Adds overall styling clean ups to our app

## How should it be tested in the UI?

- CD into the repo
- npm start
- check everything works and is reactive on all devices in the dev tools.
